folium/features.py

```diff
@@ -758,13 +758,18 @@ class MarkerCluster(Layer):
         Whether the Layer will be included in LayerControls
 
     """
-    def __init__(self, name=None, overlay=True, control=True):
+    def __init__(self, name=None, overlay=True, control=True, icon_create_function=""):
```

Comment:
please change  `icon_create_function=""`  to `icon_create_function=''` . Sounds silly, but with all the jinja templates represent JSONs it is nice to have single quotes for python code and double for JSON. (Makes it easier to read.)

Reply:
Ok, I try to do all you've said.